accepted/future-releases/enhanced-enums/feature-specification.md

```diff
@@ -213,7 +258,7 @@ enum Plain {
 }
 ```
 
-has corresponding class declaration:
+has corresponding class desugaring similar to:
```

Comment:
The terminology used previously was simply that there is "corresponding class", so maybe

```suggestion
has a corresponding class which is approximately the following:
```

Reply:
I actually changed the phrasing generally (I hope).
There is no "corresponding class".
The `enum` declaration introduces a (semantic) *class* directly.

The corresponding class desugaring is a *suggested* implementation strategy, but is not normative.